pootle/templates/browser/_actions.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 <ul>
+  {% if not can_suggest_only %}
```

Comment:
What I was trying to say is that

`not can_suggest_only`
equals to
`not can_suggest or can_translate or can_review`

While it might work ok, what we actually want is to display this only to users with `can_translate or can_review` since we don't care about whether they have or not `can_suggest`.